src/lib/openai.js

```diff
@@ -0,0 +1,47 @@
+import { OpenAI } from 'openai';
+
+function openAiConfig() {
+  const apiKey = process.env.REACT_APP_OPENAI_API_KEY;
+  if (!apiKey) {
+    throw new Error('Missing OpenAI API key in environment variables');
+  }
+  return {
+    apiKey,
+    dangerouslyAllowBrowser: true,
+  };
+}
+
+const SYSTEM_MESSAGE =
+  'You are an illustration generator that creates cartoon-style 2D digital characters based on a set of provided attributes. Always produce a clean, bold, flat-color cartoon with thick outlines, expressive facial features, and a playful aesthetic. The character must embody all attributes given (such as hairstyle, skin color, weight, height, clothing, props, emotions, and actions). Show the full character in a neutral background, posed naturally to match the description. Ensure proportions and details clearly reflect the attributes in a fun, exaggerated cartoon style';
+
```

Comment:
the images I was able to generate did not show full view of the character and also introduced artifacts of the attributes. find out what an updated prompt can be for that desired output of full body being shown and don't show attributes in the image that is generated 
<img width="1792" height="1024" alt="img-IlwHEracFYhOqt8kJ8AU8Z4g" src="https://github.com/user-attachments/assets/4c3db652-9c0a-4ff8-948b-c46f6feb876d" />